src/Glob.php

```diff
@@ -61,7 +61,7 @@ protected static function systemGlob($pattern, $flags)
                 self::GLOB_NOSORT   => GLOB_NOSORT,
                 self::GLOB_NOCHECK  => GLOB_NOCHECK,
                 self::GLOB_NOESCAPE => GLOB_NOESCAPE,
-                self::GLOB_BRACE    => GLOB_BRACE,
+                self::GLOB_BRACE    => defined('GLOB_BRACE') ? GLOB_BRACE : 0,
```

Comment:
Oh, so you land all PRs just before releasing the version? I'm used to get them landed and wait 'til the next release, thats why I've requested you to notice me. :+1: